.github/workflows/build-ddev.yml

```diff
@@ -80,9 +80,9 @@ jobs:
           os: windows-2022
         # macOS
         - target: aarch64-apple-darwin
-          os: macos-12
+          os: macos-13
```

Comment:
have you tried with 15?

Reply:
@FlorentClarret I haven't tried 15 - the reasoning for upgrading to 13 rather than jumping directly to 15 is that we want to minimize the chance of compatibility problems with 13.